sdcm/nemesis.py

```diff
@@ -242,7 +243,9 @@ def get_list_of_methods_compatible_with_backend(
 
     def _is_it_on_kubernetes(self) -> bool:
         if hasattr(self.tester, 'db_cluster'):
-            return 'kube' in type(self.tester.db_cluster).__name__.lower()
+            if not self.tester.db_cluster:
+                return False
+            return isinstance(self.tester.db_cluster, KubernetesCluster)
         return False
```

Comment:
can be simplified to

```python
return isinstance(getattr(self.tester, "db_cluster", None), KubernetesCluster)
```

or, if you prefer to have little bit more readable code:

```python
if db_cluster := getattr(self.tester, "db_cluster", None):
    return isinstance(db_cluster, KubernetesCluster)
return False
```

Reply:
fixed